lib/jsdom/living/node.js

```diff
@@ -191,3 +191,16 @@ module.exports.listOfElementsWithNamespaceAndLocalName = (namespace, localName,
     } });
   });
 };
+
+// https://dom.spec.whatwg.org/#converting-nodes-into-a-node
+// create a fragment (or just return a node for one item)
+exports.createNode = (document, nodes) => {
+  if (nodes.length === 1) {
+    return nodes[0] && nodes[0].nodeType ? nodes[0] : document.createTextNode(String(nodes[0]));
```

Comment:
ok I do it, but I wonder how element.after(2) or element.after({}) will work (it should add '2' and '[object Object]' respectively